plugins/@grouparoo/mailchimp/src/initializers/plugin.ts

```diff
@@ -27,15 +33,27 @@ export class Plugins extends Initializer {
   }
 
   async initialize() {
+    const defaultSyncMode: DestinationSyncMode = "sync";
+
     plugin.registerPlugin({
       name: packageJSON.name,
       icon: "/public/@grouparoo/mailchimp/mailchimp.svg",
       templates: [
         MailchimpAppTemplate,
         MailchimpSourceTemplate,
         MailchimpPropertyTemplate,
-        MailchimpEmailDestinationTemplate,
-        MailchimpIdDestinationTemplate,
+        new DestinationTemplate(
+          "mailchimp:email",
+          [path.join(templateRoot, "destination", "email", "*.template")],
+          emailSupportedSyncModes,
+          defaultSyncMode
+        ),
+        new DestinationTemplate(
+          "mailchimp:id",
+          [path.join(templateRoot, "destination", "id", "*.template")],
+          idSupportedSyncModes,
+          defaultSyncMode
```

Comment:
`defaultSyncMode` value is `sync`, but `id` does not support it (only enrich). You can remove `defaultSyncMode` in this case. If there is only one supported sync mode, it will use that as default.